konf/site.yaml

```diff
@@ -2,7 +2,9 @@ domain: jp.ubuntu.com
 
 image: prod-comms.docker-registry.canonical.com/jp.ubuntu.com
 
-useProxy: false
+useProxy: true
+
+readinessPath: "/"
```

Comment:
Since you're moving this line anyway, I feel you might as well delete it. Here's my thinking:

The default `readinessPath` should be `/_status/check` I think. I don't see why jp should diverge from this default. I would imagine this was originally here because JP used to be a static site, and didn't have a specific healthcheck endpoint.

Unless you know of a reason why this override is here, could we try removing it, to fall back to the default?